docs/semgrep-appsec-platform/jira.md

```diff
@@ -111,6 +109,18 @@ Repeat these steps for each mapping you want to create. When done, click **Save
 Ensure a 1:1 mapping between the Jira issue type field values and the Semgrep values.
 :::
 
+#### Multiple labels
+
+You can map multiple labels when creating a field mapping. In the **Add mapping** dialog:
```

Comment:
```suggestion
You can map multiple labels to a single Semgrep field when creating a field mapping. In the **Add mapping** dialog:
```

I don't know if this change is helpful, but I was a bit confused about the labels (this could be because I'm unfamiliar with Jira). If you don't think this suggestion is helpful, please feel free to disregard